NOTICE.txt

```diff
@@ -1,6 +0,0 @@
-SonarQube C# Plugin
```

Comment:
given we have this notice in all our repos, let's revert this until we clarify if it's needed.
I will ask around